.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+
+matrix:
+  include:
+  - go: 1.13.x
+    env: GO111MODULE=on
+  - go: 1.12.x
+    env: GO111MODULE=on
+    
+install:
+- go get github.com/mattn/goveralls
```

Comment:
Yes, it is and this is exactly what I would like to achieve. Look at this PR from my previous project as an example https://github.com/aerogear/mobile-security-service/pull/45 . Coveralls can be setup to comment in the PR, it will also be visible in the PR checks.